src/DataStructures/DataBox/DataBoxTag.hpp

```diff
@@ -435,8 +473,8 @@ struct remove_tag_prefix_impl<
 /// Wrap `Tag` in `Prefix<_, Args...>`, also wrapping variables tags
 /// if `Tag` is a `Tags::Variables`.
 template <template <typename...> class Prefix, typename Tag, typename... Args>
-using add_tag_prefix = typename detail::add_tag_prefix_impl<
-    tt::is_a_v<Tags::Variables, Tag>>::template f<Prefix, Tag, Args...>;
+using add_tag_prefix =
+    Prefix<detail::dispatch_add_tag_prefix_impl<Prefix, Tag, Args...>, Args...>;
```

Comment:
Why not just inline the `dispatch_add_tag_prefix_impl`?

Reply:
It's recursive.

Reply:
Ahh, sorry, yes, that makes sense.